x-pack/plugins/stack_connectors/public/connector_types/inference/params.tsx

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect } from 'react';
+import {
+  JsonEditorWithMessageVariables,
+  type ActionParamsProps,
+} from '@kbn/triggers-actions-ui-plugin/public';
+import { EuiTextArea, EuiFormRow, EuiSpacer, EuiSelect } from '@elastic/eui';
+import { RuleFormParamsErrors } from '@kbn/alerts-ui-shared';
+import {
+  ChatCompleteParams,
+  RerankParams,
+  SparseEmbeddingParams,
+  TextEmbeddingParams,
+} from '../../../common/inference/types';
+import { DEFAULTS_BY_TASK_TYPE } from './constants';
+import * as i18n from './translations';
+import { SUB_ACTION } from '../../../common/inference/constants';
+import { InferenceActionConnector, InferenceActionParams } from './types';
+
+const InferenceServiceParamsFields: React.FunctionComponent<
```

Comment:
I wanted to test these but couldn't find them in Kibana 😅 

Normally I'd create a rule and go to the actions but in this case I couldn't find the connector. How does this work?